scrnaseq.Rmd

```diff
@@ -1267,8 +1433,13 @@ if (length(levels(sc$orig.ident)) > 1) {
 # Choose the number of PCs to use for clustering
 sc = Seurat::FindNeighbors(sc, dims=1:param$pc_n, verbose=FALSE)
 
+```
+
+```{r clustering_resolution_test, warning=FALSE, fig.height=5, fig.width=12}
 # Seurat vignette suggests resolution parameter between 0.4-1.2 for datasets of about 3k cells
-sc = Seurat::FindClusters(sc, resolution=param$cluster_resolution, algorithm=4, verbose=FALSE, method="igraph")
+for (i in param$cluster_resolution_test) {
```

Comment:
We would like to adapt the concept, but change the code a bit. We can do that when we merge the PR. Ok?

Reply:
Ok, you are welcome to change it.